src/app/shared/components/loading-overlay/loading-overlay.component.less

```
@@ -0,0 +1,140 @@
+  -webkit-backdrop-filter: blur(6px);
+}
+
+// Frosted card that holds all elements
+.loading-card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 20px;
+  padding: 40px 48px;
+  background: transparent;
+  // border: 1px solid var(--color-primary-100);
```

Stylelint (17.3.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)